app/broadcast_areas/__init__.py

```diff
@@ -65,7 +68,15 @@ def __init__(self, row):
         self.items = BroadcastAreasRepository().get_all_areas_for_library(self.id)
 
     def get_examples(self):
-        return BroadcastAreasRepository().get_library_description(self.id)
+        # we show up to four things. three areas, then either a fourth area if there are exactly four, or "and X more".
+        areas_to_show = [area[1] for area in sorted(self.items, key=itemgetter(1))[:4]]
+
+        count_of_areas_not_named = len(self.items) - 3
+        # if there's exactly one area not named, there are exactly four - we should just show all four.
+        if count_of_areas_not_named > 1:
+            return f'{", ".join(areas_to_show[:3])} and {count_of_areas_not_named} more…'
+        else:
+            return formatted_list(areas_to_show, before_each='', after_each='')
```

Comment:
Think it’s cleaner if you always use `formatted_list` to do the concatenation:
```suggestion
        if count_of_areas_not_named > 1:
            areas_to_show = areas_to_show[:3] + [f'{count_of_areas_not_named} more…']

        return formatted_list(areas_to_show, before_each='', after_each='')
```